src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -425,6 +425,46 @@ public void PutGetPatchResponseValidation()
             var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "swagger-putgetpatch-response-validation.json"));
             messages.AssertOnlyValidationMessage(typeof(PutGetPatchResponseValidation), 1);
         }
+
+        [Fact]
+        public void RequiredReadOnlyPropertiesValidation()
```

Comment:
Instead of numbering, just mention which property you are looking for, context is always helpful